bugwarrior/services/redmine.py

```diff
@@ -82,10 +191,21 @@ def get_issue_url(self):
             self.origin['url'] + "/issues/" + six.text_type(self.record["id"])
         )
 
+    def get_converted_hours(self, estimated_hours):
+        tw = TaskWarriorShellout()
+        calc = tw._execute('calc', estimated_hours)
+        return (
+            calc[0].rstrip()
+        )
```

Comment:
None of the other services convert to taskwarrior format so I'm doubtful we should be here. You want to use the redmine duedate as your taskwarrior duedate though, and I'm not sure how, or if it's possible, to do this if the due dates are stored in integer format.